src/plotSearchAndCompetitionsPage/plotSearchAndCompetitionsPage.tsx

```diff
@@ -146,13 +157,16 @@ const PlotSearchAndCompetitionsPage = (props: Props): JSX.Element => {
         selectedTarget={selectedTarget}
         isOpen={isSidebarOpen}
         toggle={setSidebarOpen}
+        addFavouriteTarget={addFavouriteTarget}
```

Comment:
Not sure if there's a reason to pass this dispatcher from this level since it can just as well be connected in `MapSearchComponent` directly. (As opposed to some of this other stuff that's managed in this component rather than in Redux state.)